Content.Server/Body/Systems/BloodstreamSystem.cs

```diff
@@ -458,7 +458,7 @@ public void ChangeBloodReagent(EntityUid uid, string reagent, BloodstreamCompone
     /// <summary>
     ///     Stops the bleeding of the devoured target so the dragon doesn't leave a trail of random blood puddles behind
     /// </summary>
-    private void OnDevoured(EntityUid uid, BloodstreamComponent component, OnDevouredEvent args)
+    private void OnDevoured(EntityUid uid, BloodstreamComponent component, ref OnDevouredEvent args)
```

Comment:
Entity<BloodstreamComponent> ent and use ent.Comp to get the component, do this for all stuff